samples/BoltOn.Samples.WebApi/appsettings.Development.json

```diff
@@ -5,5 +5,19 @@
       "System": "Information",
       "Microsoft": "Information"
     }
+  },
+  "CosmosSettings": {
+    "CosmosDbs": {
+      "College": {
+        "Uri": "https://engagedb.documents.azure.com:443/",

```

Comment:
remove these values. when somebody wants to run the sample, they have to set their configuration and test it

Reply:
@gokulm : will remove while merging , will keep it until i test them .